README.md

```diff
@@ -1,34 +1,32 @@
 # Event Driven Ansible Server
 
+## Installation
+For instructions on how to quickly spin up a preview instance of *eda-server*, please see [INSTALL.md](./INSTALL.md).
+
 ## Setting up a development environment
 
+The below instructions detail how to setup a development environment for _eda-server_. These instructions should be performed on your local system and not on a remote system.
```

Comment:
Why shouldn't you be able to use a remote machine as a development environment? I think this sentence is not necessary.

Reply:
The task that builds and starts the UI runs `npm run start:dev`, which opens up a local browser window to the eda page and is only accessible from localhost. @ddonahue007 may be able to elaborate on this further.

Reply:
That's doesn't prevent you from using a remote system.

Reply:
When doing hot reload for UI dev work you needs to be running locally.  There are ways to get around this but to be honest I am not sure why you would want to.

Reply:
Would it be better to remove this sentence and add a note to step 5 to say this command must be run within `graphical.target`?

Reply:
I think that we just remove the sentence, If some hits an issue we can address it then I think